woocommerce/woocommerce
# [CYS] Ensure the CYS task is marked as complete only if the user customized their theme in the editor

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Ensure the CYS task is marked as complete only if the user customized their templates or global styles in the editor.



Closes #45784



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure you have the WooCommerce Beta Tester plugin installed and activated
2. Under Tools > WCA Test Helper > Features, make sure customize-store is enabled
3. Under Tools > WCA Test Helper > Tools, click on Reset Customize Your Store
4. Access WooCommerce > Home and make sure the CYS task is not marked as complete:

<img width="755" alt="Screenshot 2024-03-21 at 16 02 51" src="https://github.com/woocommerce/woocommerce/assets/15730971/1312aa68-04db-46ac-95c8-ba582403fda4">

5. Make sure you have the TT4 theme enabled or any other block theme
6. Head over to Appearance > Editor > Styles and change to a different style combination.
7. Go back to WooCommerce > Home and ensure the task is now marked as complete
8. Under Tools > WCA Test Helper > Tools, click on Reset Customize Your Store again
9. Access WooCommerce > Home one more time and make sure the CYS task is not marked as complete anymore:

<img width="755" alt="Screenshot 2024-03-21 at 16 02 51" src="https://github.com/woocommerce/woocommerce/assets/15730971/1312aa68-04db-46ac-95c8-ba582403fda4">

10. Head over to Appearance > Editor > Templates and select one template, e.g. Product Catalog, and modify it
11. Go back to WooCommerce > Home and ensure the task is now marked as complete:

<img width="1871" alt="Screenshot 2024-03-21 at 16 08 59" src="https://github.com/woocommerce/woocommerce/assets/15730971/f3c10891-cf47-4a34-8e9c-13f1c2c464c5">



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Ensure the CYS task is marked as complete only if the user customized their templates or global styles in the editor

#### Comment 

</details>
